Softeq.XToolkit.WhiteLabel.Droid/MainApplicationBase.cs

```diff
@@ -75,7 +76,16 @@ protected virtual void InitializeWhiteLabelRuntime()
             PlatformProvider.Current = new DroidPlatformProvider();
 
             // Init dependencies
-            Bootstrapper.Initialize();
+            var bootstrapper = CreateBootstrapper();
+            var container = bootstrapper.Initialize();
+            Dependencies.Initialize(container);
```

Comment:
Also, need to add `Dependencies.Initialize(container);` to Xamarin.Forms:
https://github.com/Softeq/XToolkit.WhiteLabel/blob/master/Softeq.XToolkit.WhiteLabel.Forms/FormsApp.cs

Reply:
Indeed, done. I've also updated OnStarted() callback to accept configured container